tests/telemetry/test_telemetry.py

```diff
@@ -281,14 +281,20 @@ def test_on_change_updates(duthosts, enum_rand_one_per_hwsku_hostname, ptfhost,
     logger.info("Testing on change update notifications")
 
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    if duthost.is_supervisor_node():
+        pytest.skip(
+            "Skipping test as no Ethernet0 frontpanel port on supervisor")
     skip_201911_and_older(duthost)
     cmd = generate_client_cli(duthost=duthost, gnxi_path=gnxi_path, method=METHOD_SUBSCRIBE,
```

Comment:
For multi asic support we will need to modify query for STATE_DB such that we specify asic target DB as well.

For example, if we are going to test on asic0, STATE_DB needs to be modified to use STATE_DB/asic0

Reply:
> For multi asic support we will need to modify query for STATE_DB such that we specify asic target DB as well.
> 
> For example, if we are going to test on asic0, STATE_DB needs to be modified to use STATE_DB/asic0

Yes. We need to support namespace in multi-asic scenario. I have added namespace support for generate_client_cli and tested this test_on_change_updates. We can get a successful callback now. Without this change found GNMI container crashes and restarts which also make test passed because of no callback is triggered with that scenario. I have created issue 20510 for that crash.